apps/web/pages/event-types/[type].tsx

```diff
@@ -407,6 +407,33 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
             </div>
           </div>
         );
+      case LocationType.UserPhone:
+        return (
+          <div>
+            <label htmlFor="phonenumber" className="block text-sm font-medium text-gray-700">
+              {t("set_your_phone_number")}
+            </label>
+            <div className="mt-1">
+              <input
```

Comment:
Maybe we could use the already existing component for phone input here? [Link to PhoneInput component](https://github.com/calcom/cal.com/blob/feat/phone-location/apps/web/components/ui/form/PhoneInput.tsx)